msteams-platform/m365-apps/actions-in-m365-quick-start-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Actions in Microsoft 365 quick start guide
+description: In this article, learn how to create Actions in Microsoft 365, function of Actions and its use cases.
+ms.date: 12/8/2023
+ms.author: mosdevdocs
+author: v-preethah
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365 quick start guide
+
+Get started with Actions in Microsoft 365 using the [Microsoft 365 sample](https://github.com/OfficeDev/m365-msteams-actions-preview/tree/main) [Need to update the public link.].
+
+## Prerequisites
+
+Before you get started, ensure that you install the following tools:
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | [Teams Toolkit](../toolkit/install-Teams-Toolkit.md) | A Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use the latest version. |
```

Comment:
This should probably listed next to, but after, the prereq for VS Code. You need to install the client application in order to install extensions for it.